modules/telemetry/class-vip-workflow-tracker.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace VIPWorkflow\Modules\Telemetry;
+
+use Automattic\VIP\Telemetry\Tracks;
+
+class Tracker {
+	/**
+	 * Tracks instance
+	 *
+	 * @var Tracks
+	 */
+	public Tracks $tracks;
+
+	/**
+	 * Constructor
+	 *
+	 * @param Tracks $tracks The Tracks instance
+	 */
+	public function __construct( Tracks $tracks ) {
+		$this->tracks = $tracks;
+	}
+
+	/**
+	 * Record an event
+	 *
+	 * @param string $event_name The event name
+	 * @param array $event_data The event data
+	 */
+	public function record_event( string $event_name, array $event_data = [] ): void {
+		$this->tracks->record_event( $event_name, $event_data );
```

Comment:
Does this all work properly outside of a VIP site? I see there's a class check in [`start_tracking`](https://github.com/Automattic/vip-workflow-plugin/blob/8c7f9ded80565a9e764bc7041c0db0357c002fbb/class-workflow.php#L389), but how do event calls work on a non-VIP site?

Reply:
Unless they've installed the VIP MU-plugins on their non-VIP site then the `Telemetry::init()` won't be called in `start_tracking` due to the class check you've mentioned. If they did install VIP MU-plugins on their non-VIP site, then events should still be recorded but the user ID and user ID type would be different as those depends on VIP env constants